kentikapi/models/convertible_bool.go

```diff
@@ -0,0 +1,9 @@
+package models
+
+type Bool bool
+
+func (bit *Bool) UnmarshalJSON(b []byte) error {
+	txt := string(b)
+	*bit = Bool(txt == "1" || txt == "true")
```

Comment:
you can also use strconv.ParseBool that covers a number of stringy bool values

Reply:
Please rebase onto master and use/modify BoolAsString